ontopy/ontology.py

```diff
@@ -1217,3 +1232,31 @@ def new_entity(self, name, parent):
         with self:
             entity = types.new_class(name, (parent,))
         return entity
+
+
+class BNode:
+    """Represents a blank node."""
+
+    def __init__(self, onto: Union[World, Ontology], storid: int):
+        """Initiate a blank node.
+
+        Args:
+            onto: Ontology or World instance.
+            storid: The storage id of the blank node.
+        """
+        if storid >= 0:
+            raise ValueError(
+                f"bnode is supposed to have a negative storid: {storid}"
+            )
+        self.onto = onto
+        self.storid = storid
+
+    def __repr__(self):
+        return repr(f"_:b{-self.storid}")
```

Comment:
So, here you're representing it with the positive integer. I.e., a blank node with `storid` of `-5` becomes `"_:b5"`?

Reply:
Yes, that is intentional, "_:b-5" would be ugly and could cause problems for some tools. 

bnodes have no public IRI, so it doesn't matter how we choose to represent as long we are internally consistent.